Configuration/StandardSequences/python/PATMC_cff.py

```diff
@@ -13,3 +13,9 @@
 )
 
 miniAOD=cms.Sequence()
+
+patGENTask = cms.Task(
+    slimmingGENTask
+)
+
+miniGEN=cms.Sequence()
```

Comment:
can we go a step further and complete the decoupling (most importantly so that the totality of miniAOD config is not loaded for PATGEN).
Create PATGEN_cff with 
```python
 import FWCore.ParameterSet.Config as cms
 from PhysicsTools.PatAlgos.slimming.genParticles_cff import *
patGENTask = cms.Task( 
   genParticlesTask
}
miniGEN=cms.Sequence() 
```
this seems to be enough for the purpose